index.bs

```diff
@@ -424,58 +433,150 @@ When this method is invoked, the user agent MUST execute the following algorithm
-    from the authenticators:
-    - If the |adjustedTimeout| timer expires, then for each entry in |issuedRequests| invoke the <a>authenticatorCancel</a>
-        operation on that authenticator and remove its entry from the list.
-    - If any authenticator returns a status indicating that the user cancelled the operation, delete that authenticator's
-        entry from |issuedRequests|. For each remaining entry in |issuedRequests| invoke the <a>authenticatorCancel</a>
-        operation on that authenticator and remove its entry from the list.
-    - If any authenticator returns an error status, delete the corresponding entry from |issuedRequests|.
-    - If any authenticator indicates success:
-        - Remove this authenticator's entry from |issuedRequests|.
-        - Create a new {{ScopedCredentialInfo}} object named |value| and populate its fields with the values returned from the
-            authenticator as well as the {{ScopedCredentialInfo/clientDataJSON}} computed earlier.
-        - For each remaining entry in |issuedRequests| invoke the <a>authenticatorCancel</a> operation on that authenticator and
-            remove its entry from the list.
-        - Resolve |promise| with |value| and terminate this algorithm.
+    <dt>If any |authenticator| returns a status indicating that the user cancelled the operation,</dt>
 
-1. Reject |promise| with a {{DOMException}} whose name is "{{NotAllowedError}}", and terminate this algorithm.
+    <dd>
+        1. [=set/Remove=] |authenticator| from |issuedRequests|.
+        2. [=set/For each=] remaining |authenticator| in |issuedRequests| invoke
+            the <a>authenticatorCancel</a> operation on |authenticator| and
+            [=set/remove=] it from |issuedRequests|.
+
+    </dd>
+
+    <dt>If any |authenticator| returns an error status,</dt>
+
+    <dd>[=set/Remove=] |authenticator| from |issuedRequests|.</dd>
+
+    <dt>If any |authenticator| indicates success,</dt>
+
+    <dd>
+        1. [=set/Remove=] |authenticator| from |issuedRequests|.
+        2. Let |value| be a new {{ScopedCredentialInfo}} object whose fields are:
+            : {{ScopedCredentialInfo/clientDataJSON}}
+            :: A new {{ArrayBuffer}} containing the bytes of |clientDataJSON|.
+            : {{ScopedCredentialInfo/attestationObject}}
+            :: A new {{ArrayBuffer}} containing the bytes of the value returned
+                from the successful [=authenticatorMakeCredential=] operation
+        3. [=set/For each=] remaining |authenticator| in |issuedRequests| invoke the
+            <a>authenticatorCancel</a> operation on |authenticator| and
+            [=set/remove=] it from |issuedRequests|.
+        4. [=Resolve=] |promise| with |value| and terminate this algorithm.
+
+    </dd>
+
+1. [=Reject=] |promise| with a {{DOMException}} whose name is
+    "{{NotAllowedError}}".
+
+    Issue: {{NotAllowedError}} seems incorrect for at least the timeout and
+    cancelled exit conditions above.
```

Comment:
Suggested alternative? If you have one we can just put it in with this PR or open a new issue.